### ISSUE TITLE
cannot initialize using node

### ISSUE THREAD

**Issue by danidev** [gitbrent/PptxGenJS#199]
hello, 
I'm using PptxGenJS on a node/express application.
If I use
`var pptx = require('pptxgenjs');`
in several requests, the ppt is not initialized and adds slides on the same instance. 
I saw that in client is possible using new keyword on every instance.
I cannot use the constructor, this doesn't work:
`var pptxInstance = new pptx();`
How can I create a new instance on every request?
Thanks,
Daniele

**Comment by gitbrent** [comment]
Hi @danidev ,

There's a section in the [README](https://github.com/gitbrent/PptxGenJS#saving-multiple-presentations) that points to Issue #83.  Check there to see what the current status of this is.

**Comment by danidev** [comment]
thanks @gitbrent 
I've found the work-around and I did what suggested by the thread:
```
var pptxgenjs = require('pptxgenjs');
var pptx = new pptxgenjs.constructor();
```
Anyway I think it will be better having an initialization function. 
if it is a singleton as I read somewhere, it could be an initialize() function without calling the constructor that breaks the pattern usage.
If it is a "static" class, could be a getNewInstance() that return a new object without calling explicitly the constructor.
